### ISSUE TITLE
Incorrect handling of MIDI sustain pedal controller.

### ISSUE THREAD

**Issue by andy-fillebrown** [csound/csound#815]
When using a MIDI file for the score, sustain pedal release events turn off all notes on the same channel whether the note is currently being held or not. Notes currently being held should not turn off when the sustain pedal is released.

Considering this sequence of events...

TIME / EVENT
----------------
1.10  / pedal down
1.12  / note 60 on
1.25  / pedal up
1.50  / note 60 off

The note will be turned off at 1.25 instead of 1.50.

Currently, the only workaround is to read and parse all MIDI events manually using the midiin opcode in an always-on instrument and go from there. Fortunately, I'm using FluidSynth for my current project, so I can just pass the MIDI data directly to the fluidControl opcode.

**Comment by jpffitch** [comment]
Any developer understand where pedals are sensed?

**Comment by vlazzarini** [comment]
I think this is cc 64, so we need to look in the incoming midi data path somewhere if cc 64 is looked at.

**Comment by andy-fillebrown** [comment]
Thanks for looking into this. Please note that this issue might be specific to MIDI *file* input. Realtime MIDI input might not have this issue.

**Comment by andy-fillebrown** [comment]
I made a test case but I haven't been able to reproduce the issue using Csound 6.13.0 or Csound 6.09.1. The project I was working on at the time was running into this issue, but the sequence of events I listed when opening the issue don't reproduce the issue on the new version of Csound or the version of Csound I was working with at the time. I'll go ahead and close this issue if nobody objects. If I run into the bug again sometime I'll either reopen this issue or make a new one.
